depthai_bridge/src/ImageConverter.cpp

```diff
@@ -142,22 +129,20 @@ void ImageConverter::toRosMsg(std::shared_ptr<dai::ImgFrame> inData, std::deque<
         switch(inData->getType()) {
             case dai::RawImgFrame::Type::RGB888p: {
                 cv::Size s(inData->getWidth(), inData->getHeight());
-                std::vector<cv::Mat> channels;
-                // RGB
-                channels.push_back(cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 2));
-                channels.push_back(cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 1));
-                channels.push_back(cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 0));
-                cv::merge(channels, output);
+                cv::Mat m1 = cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 2);
+                cv::Mat m2 = cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 1);
+                cv::Mat m3 = cv::Mat(s, CV_8UC1, inData->getData().data() + s.area() * 0);
+                cv::Mat channels[3] = {m1, m2, m3};
+                cv::merge(channels, 3, output);
```

Comment:
Is this faster ? 
Also if we can reserve and load like the vectors it might be even faster ?

Reply:
We could try reserving, although I'm not sure if we gain anything by using vector here.

Currently callbacks look like this:

Without optimizations:
![image](https://github.com/luxonis/depthai-ros/assets/11968712/db50c7ec-f221-4c02-9575-cc822a7b1bec)
With optimizations:
![image](https://github.com/luxonis/depthai-ros/assets/11968712/3a95a2cf-df36-4c34-b982-5a89dfbedccf)